modules/experimental/pages/psychic-focus.adoc

```diff
@@ -0,0 +1,15 @@
+= Psychic Focus
+
+`Detachment ability`
+
+Some detachments contain many low-level Psykers, not powerful enough to have the usual effect in an assault.
+---
+
+Where your detachment has the {psychic-focus} ability:
+
+* The detachment has a free Psychic Blast card with a value of 0, that can only be used by this detachment.
+* Psychic Fate Cards can be played from any unit in the detachment, not just from {psyker} units.
```

Comment:
Do we want to convey that the player should (1) actually have a physical card, (2) act in all ways as if they have a physical card, or (3) gain the equivalent as if they had a card?

Each has a slightly different implication:

(1) is obviously awkward for obvious practical reasons of supply, and also some overlap with reasons in point (2).
(2) is slightly awkward in as much as any effect that targets Fate cards (eg, 'discard a random Fate card' or some such[1]) would necessitate some other method of selection, such as a die roll. Plus it is arguably a bit odd given that the card is specific to this detachment and therefore not quite the same as a regular Fate card anyway.
(3) is easiest since it doesn't have any of the above problems. It'd imply immunity to any effect that targets Fate cards[1], but given its limitation I think that'd be okay and perhaps preferable.

_[1]Perhaps there are no such effects yet, I can't recall — but you see my overall point, I'm sure._

Assuming (3) wins, I suggest something like this:

```suggestion
* Once per game, you can make a special psychic attack with this detachment.
This works exactly as if it were a Fate card with the title 'Psychic Attack (0) — Psychic Blast'.
* When you wish to make the above attack, or play a Psychic Attack Fate card, you may choose any unit in this detachment to channel the effects, as if that unit had the {psyker} ability.
```

Reply:
And another related thought: It seems sensible that these units can't channel a 'nullify psychic attack' effect like a _Psyker_ unit can, because they aren't powerful enough. Assuming that this is indeed what you intend, perhaps my second bullet in the above suggestion will need some more work to make it 100% clear that the use of a Fate card to nullify an attack is not allowed here?

Reply:
It's option (3) for the free attack, although for practical purposes I've been keeping a token with the detachment until they've used the attack.

For the related thought, the intent is that _any_ type of Psychic Fate card can be played from _any_ unit in the detachment. So yes, they can use a Fate card to nullify an attack. Although that wouldn't matter very often as it would only kick in if you had no actual _Psyker_ units left on the table!

Reply:
> the intent is that any type of Psychic Fate card can be played from any unit in the detachment. So yes, they can use a Fate card to nullify an attack.

Ah, okay. I found it a bit ambiguous when I was looking at the _Psyker_ ability text, and thought perhaps you'd left that bit out on purpose — but when I look at the Psychic Blast card text, it's less ambiguous because both the above and the nullify text involve the term 'play' in relation to the action with the Fate card.

I'll probably apply my suggestion shortly, but then think about whether there's a little more to do to remove ambiguity. It might come out in the wash in due course anyway, because we'll want to tweak the _Psyker_ ability text at some point to refer to Psychic Attack cards in general, rather than Psychic Blast in particular.